### PR TITLE
Add OAUTH docs

### PR DESCRIPTION
**Pull request by Philinphiladelphia**
## Issue Number

N/a

## Purpose/Implementation Notes

This is a brain drain on all the relevant info I have on OAUTH.

Where should this be located? Should it be where it is in the file structure?

## Types of changes

- New feature (non-breaking change which adds functionality)

## Functional tests

None

## Checklist

- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: api/resources_portal/OAUTH_README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+# Email Visibility
+As part of our account creation, we attempt to access the user's email from their ORCID record. There are three possible visibilities for their email:
+1. Public
+2. Trusted Parties
+3. Private
+
+If the user's email is set to Public or Trusted Parties, we can access it. If it's Private, we cannot.
+
+## Sandbox vs Production
+ORCID supports a sandbox environment for testing and staging. The only differences between Sandbox and Prod are seperate databases and the fact that Sandbox only support email addresses from [mailinator.com], a website which allows you to access public email addresses.
```
Comment (kurtwheeler) [inline api/resources_portal/OAUTH_README.md:35]:
```suggestion
ORCID supports a sandbox environment for testing and staging. The only differences between Sandbox and Prod are seperate databases and the fact that Sandbox only support email addresses from [mailinator.com](mailinator.com), a website which allows you to access public email addresses.
```

File: api/resources_portal/OAUTH_README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+Account 1:
+| Email    | rportaltest@mailinator.com
+| Password | PASSWORD1
+
+This account has its email preferences set to "Trusted Parties".
+
+Account 2:
+| Email    | rportaltest2@mailinator.com
+| Password | PASSWORD1
```
Comment (kurtwheeler) [inline api/resources_portal/OAUTH_README.md:49]: These tables are not formatted correctly. I think you need a line like
```
-----------------|--------|---
```
except with the correct number of dashes.

File: api/resources_portal/OAUTH_README.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+3. Private
+
+If the user's email is set to Public or Trusted Parties, we can access it. If it's Private, we cannot.
+
+## Sandbox vs Production
+ORCID supports a sandbox environment for testing and staging. The only differences between Sandbox and Prod are seperate databases and the fact that Sandbox only support email addresses from [mailinator.com], a website which allows you to access public email addresses.
+
+## Testing Accounts
+
+We have two established sandbox accounts which can be used for testing. More accounts can be created very easily if needed.
```
Comment (kurtwheeler) [inline api/resources_portal/OAUTH_README.md:39]: > More accounts can be created very easily if needed.

How so? Can you explain that a bit and maybe drop a relevant link or two?

**Review by kurtwheeler — APPROVED** [review]
This looks great!! It reads just like a story! I left a few comments. In addition to those, could you link to this from the primary readme? Also, could you do one sentence per line in the markdown? It makes working with it a bit easier.

File: api/resources_portal/OAUTH_README.md (hunk 3 above)
Comment (Philinphiladelphia) [reply api/resources_portal/OAUTH_README.md]: Yep! Added an explaination.

File: api/resources_portal/OAUTH_README.md (hunk 2 above)
Comment (Philinphiladelphia) [reply api/resources_portal/OAUTH_README.md]: You're right- I fixed the format.

File: api/resources_portal/OAUTH_README.md (hunk 1 above)
Comment (Philinphiladelphia) [reply api/resources_portal/OAUTH_README.md]: Done.

**Review by kurtwheeler — APPROVED** [review]
This all looks great! However I think it might be best to not nest the `.md` files. Just put them both top level!

**Pull request merged**